components/prize_pool/wikis/rainbowsix/prize_pool_legacy.lua

```diff
@@ -0,0 +1,158 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:PrizePool/Legacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Currency = require('Module:Currency')
+local Lua = require('Module:Lua')
+local Points = mw.loadData('Module:Points/data')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local CustomPrizePool = Lua.import('Module:PrizePool/Custom', {requireDevIfEnabled = true})
+
+local LegacyPrizePool = {}
+local nextPoints, nextQual, nextFreetext = 1, 1, 1
+local inputToId = {}
+local qualCache = {}
```

Comment:
don't we `_` prefix those?

Reply:
Locals don't need to be `_` prefixed, but this is wrong indeed.